Overview.src.html

```diff
@@ -3355,119 +3355,82 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
-return true if the <span title=concept-Body-used-flag>used flag</span> is set, and false
-otherwise.
+return true if <span title="concept-Body-disturbed">disturbed</span>, and false otherwise.
 
 <p>Objects implementing the <code>Body</code> mixin also have an associated
-<dfn title=concept-Body-consume-body>consume body</dfn> algorithm, which given a
-<var title>type</var>, runs these steps:
+<dfn title=concept-Body-package-data>package data</dfn> algorithm, which given
+<var>bytes</var>, a <var>type</var> and a <var>MIME type</var>, runs these steps:
 
 <ol>
- <li><p>Let <var title>p</var> be a new promise.
-
- <li><p>If <span title=concept-Body-used-flag>used flag</span> is set, reject
- <var title>p</var> with a <code title>TypeError</code>.
-
- <li>
-  <p>Otherwise, set <span title=concept-Body-used-flag>used flag</span> and then run these
-  substeps <span data-anolis-spec=html>in parallel</span>:
-  <!-- We do not check for body being null to make this API work consistently.
-       Otherwise we'd need an additional flag. -->
-
-  <ol>
-   <li><p>Let <var title>stream</var> be <span title=concept-Body-body>body</span>.
-
-   <li><p>If <var title>stream</var> is null, set <var title>stream</var> to an empty byte
-   sequence.
-
-   <li><p>Let <var title>bytes</var> be the result of reading from <var title>stream</var>
-   until it returns end-of-stream.
-   <!-- XXX stream -->
-
-   <li>
-    <p>Switch on <var title>type</var>:
-
-    <dl class=switch>
-     <dt><i title>ArrayBuffer</i>
-     <dd><p>Resolve <var title>p</var> with an <code>ArrayBuffer</code> whose contents are
-     <var title>bytes</var>.
-
-     <dt><i title>Blob</i>
-     <dd><p>Resolve <var title>p</var> with a <code data-anolis-spec=fileapi>Blob</code>
-     whose contents are <var title>bytes</var> and
-     <code data-anolis-spec=fileapi title=dom-Blob-type>type</code> is
-     <span title=concept-Body-MIME-type>MIME type</span>.
-
-     <dt><i title>FormData</i>
-     <dd>
-      <p>If <span title=concept-Body-MIME-type>MIME type</span> (ignoring parameters)
-      is `<code>multipart/form-data</code>`, run these subsubsteps:
-
-      <ol>
-       <li><p>Parse <var title>bytes</var>, using the value of the
-       `<code title>boundary</code>` parameter from
-       <span title=concept-Body-MIME-type>MIME type</span> and
-       <span data-anolis-spec=encoding>utf-8</span> as encoding, per the rules set forth
-       in <cite>Returning Values from Forms: multipart/form-data</cite>.
-       <span data-anolis-ref>RFC2388</span>
-
-       <li><p>If that fails for some reason, reject <var title>p</var> with a
-       <code title>TypeError</code>.
-
-       <li><p>Otherwise, resolve <var title>p</var> with a new
-       <code data-anolis-spec=xhr>FormData</code> object, appending each entry, resulting
-       from the parsing operation, to
-       <span data-anolis-spec=xhr title=concept-FormData-entry>entries</span>.
-      </ol>
+ <li><p>Switch on <var>type</var>:
```

Comment:
This seems to be the only step in this list? I think would be better to avoid the `<ol>` then and say something along the lines of "switch on type and run the associated steps" in the previous paragraph.